builtin/providers/aws/resource_aws_cloudfront_web_distribution_test.go

```diff
@@ -0,0 +1,118 @@
+package aws
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccCloudFront(t *testing.T) {
```

Comment:
Really a nitpick, but applying the naming convention here would make it `TestAccAWSCloudFront_basic`.